applications/CoSimulationApplication/python_scripts/data_transfer_operators/mapping.py

```diff
@@ -11,14 +11,26 @@ def Create(settings):
     return MappingDataTransferOperator(settings)
 
 class MappingDataTransferOperator(CoSimulationDataTransferOperator):
+
```

Comment:
`MappingDataTransferOperator` Why not be more specific and call it `KratosMappingDataTransferOperator`

Reply:
ok